supervisor/utils/codenotary.py

```diff
@@ -58,7 +58,7 @@ async def cas_validate(
             env=clean_env(),
         )
 
-        async with async_timeout.timeout(10):
+        async with async_timeout.timeout(15):
```

Comment:
Is 15 enough? 60 maybe?

Reply:
then in the middle 30

Reply:
it's only an CI issue it seems with coredns